DuckDuckGo/TabViewController.swift

```diff
@@ -43,12 +43,25 @@ class TabViewController: WebViewController {
         super.init(coder: aDecoder)
         webEventsDelegate = self
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addContentBlockerConfigurationObserver()
+    }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         resetNavigationBar()
     }
     
+    private func addContentBlockerConfigurationObserver() {
+        NotificationCenter.default.addObserver(self, selector: #selector(onContentBlockerConfigurationChanged), name: ContentBlockerConfigurationChangedNotification.name, object: nil)
```

Comment:
This observer never seems to get removed either.

Reply:
Ad above.